### DISCUSSION TITLE
how to understand `Dependency-free`?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [tree-sitter/tree-sitter#3470] [Q&A] [answered]
When I read [tree-sitter document](https://tree-sitter.github.io/tree-sitter/):

> Dependency-free so that the runtime library (which is written in pure [C](https://github.com/tree-sitter/tree-sitter/tree/master/lib)) can be embedded in any application

what's the meaning of this paragraph? how to understand `Dependency-free`?

**Comment by sogaiu** [comment, marked as answer]
IIUC, it refers to not requiring any external dependencies.
